packages/gatsby-plugin/src/builder-config.js

```diff
@@ -0,0 +1,25 @@
+const invariant = require(`invariant`)
+
+module.exports.defaultOptions = {
+  // Arbitrary name for the remote schema Query type
+  fieldName: `allBuilderModels`,
+  typeName: `builder`,
+}
+
+module.exports.getGQLOptions = options => {
+  const config = {
+    ...module.exports.defaultOptions,
+    ...options,
+  }
+  invariant(
+    config.publicAPIKey && config.publicAPIKey.length > 0,
+    `gatsby-plugin-builder requires a public API Key`
```

Comment:
if our plugin is called `@builder.io/gatsby` should all of these `gatsby-plugin-builder` be replaced with `@builder.io/gatsby`?